core/src/test/java/com/yahoo/ycsb/generator/AcknowledgedCounterGeneratorTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2012 YCSB contributors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+package com.yahoo.ycsb.generator;
+
+import java.util.Random;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+
+import junit.framework.TestCase;
+
+/**
+ * Tests for the AcknowledgedCounterGenerator class.
+ */
+public class AcknowledgedCounterGeneratorTest extends TestCase {
```

Comment:
Switched it but it is the TestNG annotation not the JUnit annotation. The JUnit version is 3.8.1. Something else we should consider cleaning up.